src/main/catalog-sources/kubeconfig-sync.ts

```diff
@@ -36,9 +37,23 @@ import { UserStore } from "../../common/user-store";
 import { ClusterStore, UpdateClusterModel } from "../../common/cluster-store";
 import { createHash } from "crypto";
 import { homedir } from "os";
+import globToRegExp from "glob-to-regexp";
+import { inspect } from "util";
 
 const logPrefix = "[KUBECONFIG-SYNC]:";
 
+/**
+ * This is the list of globs of which files are ignored when under a folder sync
+ */
+const ignoreGlobs = [
```

Comment:
Could you split ignore stuff to a separate PR?

Reply:
Sure